_includes/tutorials/ksql-heterogeneous-json/ksql/markup/dev/create-different-type-stream.adoc

```diff
@@ -0,0 +1,23 @@
+The first thing we do is to create a stream `DATA_STREAM` based off stock purchase transactions topic `source_data`. Within the `CREATE STREAM` statement, you'll use a `VARCHAR` keyword to define each of outer most element of the JSON types.
```

Comment:
There are a few instances of "stock purchase transactions" -- is that supposed to be there, or it's remnant from a clone?

Reply:
whoops!